crates/router/src/connector/shift4/transformers.rs

```diff
@@ -157,66 +169,61 @@ fn get_card_payment_request<T>(
     item: &types::RouterData<T, types::PaymentsAuthorizeData, types::PaymentsResponseData>,
     card: &api_models::payments::Card,
 ) -> Result<Shift4PaymentsRequest, Error> {
-    let submit_for_settlement = item.request.is_auto_capture()?;
+    let captured = item.request.is_auto_capture()?;
```

Comment:
`submit_for_settlement` gives more meaning to the variable rather than `captured`. Please keep the existing name itself